azure-functions/acbs-function/helpers/mandatoryFields.js

```diff
@@ -1,4 +1,4 @@
-const findMissingMandatory = (data, mandatoryFields) => mandatoryFields.map((mf) => (data[mf] ? '' : mf)).filter((mf) => mf);
+const findMissingMandatory = (data, mandatoryFields) => mandatoryFields.map((mf) => (data[mf] !== undefined ? '' : mf)).filter((mf) => mf !== '');
```

Comment:
`data[mf] ? '' : mf`

instead of 

`data[mf] !== undefined ? '' : mf)`

Reply:
Updated